_config.yml

```diff
@@ -9,7 +9,6 @@ federalist_component_preview: "iframe.html?id="
 federalist_release_prefix: "release-"
 touchpoints:
   active: true
-  hosted: true
 # this is for pages that don't have a permalink (primarily posts)
```

Comment:
Does this remove the option of toggling the hosted version in different environments?

Reply:
@mejiaj It does. The `hosted` option does not provide the necessary `source` data, so it is now hard-cored that the production environments have the equivalent of what was `hosted: false` and the non-production environments have the equivalent of `hosted: true`. I felt it could introduce error and confusion to retain this as a configurable option, so I removed it. 

Let me know if you think it would be helpful to keep this config.